tests/test_case_validators.py

```diff
@@ -24,17 +24,14 @@ def test_camel_cased_words():
 def test_camel_case_less_than_two_chars():
     """
     When the length of an input is less than 2, our regex logic breaks down,
-    :return:
     """
+    is_camel_case("")
```

Comment:
So one of the errors from the pytest flake8 hook is to make tests more explicit by limiting expressions inside a `pytest.raises` block to just one expression. This is a good example of why: almost none of these calls actually raise error 😄